docs/ux-manager.md

```diff
@@ -7,7 +7,11 @@ Here is a code sample and a definition of the different parameters:
 ````
 LuigiClient.uxManager().addBackdrop()
 LuigiClient.uxManager().removeBackdrop()
+LuigiClient.uxManager().showLoadingIndicator()
+LuigiClient.uxManager().hideLoadingIndicator()
 ````
 
 - **addBackdrop()** adds a backdrop to block the top and side navigation. It is based on Fundamental UI Modal, which you can use in your micro front-end to achieve the same behaviour.
-- **removeBackdrop()** removes the backdrop.
\ No newline at end of file
+- **removeBackdrop()** removes the backdrop.
+- **showLoadingIndicator()** adds a backdrop with a loading indicator for the micro front-end frame. This overrides the `loadingIndicator.enabled` setting.
```

Comment:
**loadingIndicator.enabled** -> please make it bold as with the rest of the parameters